geopandas/geodataframe.py

```diff
@@ -1852,6 +1852,81 @@ def explore(self, *args, **kwargs):
         """Interactive map based on folium/leaflet.js"""
         return _explore(self, *args, **kwargs)
 
+    def sjoin(self, df, *args, **kwargs):
+        """Spatial join of two GeoDataFrames.
+
+        See the User Guide page :doc:`../../user_guide/mergingdata` for details.
+
+        Parameters
+        ----------
+        df : GeoDataFrame
+        how : string, default 'inner'
+            The type of join:
+
+            * 'left': use keys from left_df; retain only left_df geometry column
+            * 'right': use keys from right_df; retain only right_df geometry column
+            * 'inner': use intersection of keys from both dfs; retain only
+            left_df geometry column
+
+        op : string, default 'intersects'
```

Comment:
```suggestion
        predicate : string, default 'intersects'
```

`op` was deprecated in #1626